src/components/ResetPassword/ResetPasswordModal/ResetPasswordModal.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import CommonModal from '../../../common/CommonModal/CommonModal';
+import ResetPasswordModalContent from './ResetPasswordModalContent';
+import styles from './ResetPasswordModal.module.css';
+import CheckMark from '../../../assets/BlueCheckMark.png';
+
+// TODO:
+// - Implement resetPassword
+
+const resetPassword = () => {
+  // Make request to reset password here
+  // eslint-disable-next-line no-console
+  console.log('password reset complete');
+};
+
+const ResetPasswordModal = ({ isOpen, onClose }) => {
+  const [modalStep, setModalStep] = useState(0);
```

Comment:
Do we need to use the modalStep for the ResetPasswordModal? I believe there is now only 1 step in the modal since we removed the confirmation. Correct me if I'm wrong, but we should be able to remove the modalStep from wherever it's used for this modal.